lib/smart_proxy_dynflow_core/launcher.rb

```diff
@@ -1,7 +1,7 @@
 require 'webrick/https'
 require 'smart_proxy_dynflow_core/bundler_helper'
 require 'smart_proxy_dynflow_core/settings'
-require 'smart_proxy_dynflow_core/webrick-patch'
```

Comment:
Something like the following snippet should do the trick
```ruby
require 'smart_proxy_dynflow_core/webrick-patch' unless defined?(::Proxy::Launcher)
```

We can't check if just `::Proxy` is defined because `smart_proxy_dynflow` openes that module at it will always be present

Reply:
Oh just like that, ok. Rebased.